src/app/component/user/components/profile/edit-profile/edit-profile.component.scss

```diff
@@ -211,6 +211,10 @@ h2 {
     margin-left: 100px;
     max-width: 546px;
   }
+
+  h2 {
+    font-size: 35px;
```

Comment:
how about using dynamic font size units ? see https://css-tricks.com/almanac/properties/f/font-size/

Reply:
done, changed to em